packages/firestore/lite/src/api/database.ts

```diff
@@ -41,6 +41,7 @@ import { Settings } from '../../';
 // settings() defaults:
 const DEFAULT_HOST = 'firestore.googleapis.com';
 const DEFAULT_SSL = true;
+const DEFAULT_FORCE_LONG_POLLING = false; // Used by full SDK
```

Comment:
Is this necessary given the fixes we have in the pipeline? Will you pull this out later?